notebooks/MAST/Identifying-transient-candidates/identifying_highz_candidates.ipynb

```diff
@@ -0,0 +1,598 @@
+{
```

Comment:
<p>Some comments in this code would help readers follow exactly what's going on here. For example, you might explain that since info is a list of dictionaries (or whatever crazy thing), you're adding an index to better call the data</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>